pkg/agent/qrm-plugins/cpu/dynamicpolicy/cpuadvisor/helper.go

```diff
@@ -39,23 +42,18 @@ func NewBlockCPUSet() BlockCPUSet {
 	return make(BlockCPUSet)
 }
 
-func (b BlockCPUSet) GetCPUSet() machine.CPUSet {
-	cpusets := machine.NewCPUSet()
-	for _, cpuset := range b {
-		cpusets = cpusets.Union(cpuset)
-	}
-	return cpusets
-}
-
 // GetBlocks parses ListAndWatchResponse and returns map[int]map[string]*Block,
 // the map is keyed as numa id -> block id -> block
-func (lwr *ListAndWatchResponse) GetBlocks() (map[int]map[string]*Block, error) {
+func (lwr *ListAndWatchResponse) GetBlocks() (map[int]map[string]*Block, map[int][]string, error) {
```

Comment:
why should still keep the previous returned value (i.e. `map[int]map[string]*Block`) here? just for legacy reasons or for coding convenience ?